src/frontends/common/src/input_model.cpp

```diff
@@ -33,6 +33,18 @@ Place::Ptr InputModel::get_place_by_tensor_name(const std::string& tensor_name)
     FRONTEND_RETURN_STATEMENT("get_place_by_tensor_name", m_actual->get_place_by_tensor_name(tensor_name))
 }
 
+Place::Ptr InputModel::get_place_by_input_index(size_t input_idx) const {
+    if (!m_actual) {
+        return {};
+    }
+    /*const auto& inputs = get_inputs();
+    if (input_idx < inputs.size()) {
+        return inputs.at(input_idx);
+    }
+    return {};*/
```

Comment:
```suggestion
```

Reply:
what will it return for other frontends that have no implementation?

Reply:
Will need to add that commented implementation in other frontends. There is no way to do it in base class as I don't see a way